kernel/src/process.rs

```diff
@@ -1200,15 +1003,23 @@ impl Process<'a> {
 
         // register values
         let (r0, r1, r2, r3, r12, sp, lr, pc, xpsr) = (
-            self.r0(),
-            self.r1(),
-            self.r2(),
-            self.r3(),
-            self.r12(),
-            self.sp(),
-            self.lr(),
-            self.pc(),
-            self.xpsr(),
+            // self.r0(),
+            // self.r1(),
+            // self.r2(),
+            // self.r3(),
+            5,
+            6,
+            7,
+            8,
+            9,
+            // self.r12(),
+            self.sp() as usize,
+            10,
+            11,
+            12,
+            // self.lr(),
+            // self.pc(),
+            // self.xpsr(),
```

Comment:
I see that this will all likely get fixed in #1115. Should that PR and this one be merged? Without those changes, this is a pretty bad regression for Tock debugging.

Reply:
It absolutely gets fixed.